### DISCUSSION TITLE
Question about Regression Test

### DISCUSSION THREAD

**Discussion by vladi7** [cvc5/cvc5#7386] [Q&A]
Hello,

I am really interested in the following regression test: https://github.com/CVC4/CVC4/blob/a062043b187afe410f0de3568f57594e74eb8d25/test/regress/regress2/quantifiers/net-policy-no-time.smt2

It occurs to me that this is actually a case study. I was wondering if there is a description for this somewhere that I am not able to locate? 

I am really interested in it as it seems to me as in this file, the liveness and safety properties are being checked, and that is kind of what I am working on right now. 

Thank you,
Vlad

**Comment by martin-cs** [comment]
@ajreynol added the benchmark a couple of years ago in commit https://github.com/CVC4/CVC4/pull/2126 . Unfortunately there isn't enough information in the commit log to say where it came from but hopefully Andy might remember...

**Comment by ajreynol** [comment]
Hi Vlad,

Yes, I do remember this regression.

This was a prototype encoding a possible application of SMT for network synthesis that I manually crafted.  I had been planning at some point to optimize CVC4 for these kind of problems, but I did not write this up in a paper/case study.

What sort of application are you looking at?

Cheers,
Andy

**Comment by vladi7** [comment]
Hi @ajreynol,

Thank you for your response. I am just trying to do a simple bounded model checking and SMT planning with CVC4 and looking at any code I can find. I am currently doing that with just a java loop and calling the cvc4 in each iteration, and was wondering if there is a better way?
Maybe you could provide some examples of that in plain smtlibv2 format? I would really appreciate that.

Vlad
